src/questions/derivatives.yaml

```diff
@@ -0,0 +1,948 @@
+  type: SingleChoice
+  content: What are Speculations in Derivative Market?
+  hint: NoHint
+  explanation: Speculations means using market fluctuations to gain profits in long term.
+  answerKeys:
+    - C
+  subTopics:
+    - "introduction-to-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Choosing the best asset for investment
+      key: A
+    - content: Minimise risk by offsetting position
+      key: B
+    - content: Use Market Fluctuations to gain profit
+      key: C
+    - content: None of the above
+      key: D
+- uuid: 9649c307-a437-4480-9c50-9b2d523cc099
+  type: SingleChoice
+  content: Hedgers are ..... and Speculators are .....?
+  hint: Benefits of hedging and Speculations
+  explanation: Hedgers are risk averse and speculators are risk lovers.
+  answerKeys:
+    - A
+  subTopics:
+    - "introduction-to-derivatives"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Risk Averse, Risk Lover
+      key: A
+    - content: Risk Lovers, Risk Averse
+      key: B
+    - content: Both are Risk Mitigators
+      key: C   
+- uuid: 5e6065e3-bedb-4c91-b165-8ab1d0f467c5
+  type: SingleChoice
+  content: Low liquidity in forward markets means
+  hint: NoHint
+  explanation: Low liqiuidity in forward markets refers to difficulty while transaction are taking place.
+  answerKeys:
+    - C
+  subTopics:
+    - "types-of-derivatives"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Investors face losses
+      key: A
+    - content: Conversion to cash is difficult 
+      key: B
+    - content: Transactions are not carried out easily
+      key: C
+    - content: None of the above
+      key: D
+- uuid: 82d71f82-8e88-43ff-9d13-1e3cb371d286
+  type: SingleChoice
+  content: Which of the following are the disadvantages of forward contract?
+  hint: NoHint
+  explanation: All the reasons mentioned constitute to the disadvantges of a forward contract.
+  answerKeys:
+    - D
+  subTopics:
+    - "types-of-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Finding a counterparty is difficult
+      key: A
+    - content: Low liquidity
+      key: B
+    - content: Default risk
+      key: C
+    - content: All of the above
+      key: D
+- uuid: ad133f59-6685-42fe-b47f-d2d575140e1c
+  type: SingleChoice
+  content: How are forward contracts better over futures?
+  hint: Benefits of Forward and Futures contract
+  explanation: Forward and Futures contract are identical in temrns of standardisation, default risk and they are equally liquid as well.
+  answerKeys:
+    - D
+  subTopics:
+    - "types-of-derivatives"
+  difficultyLevel: "Medium"
+  choices:
+    - content: They are standardised
+      key: A
+    - content: They have lower default risk
+      key: B
+    - content: They are more liquid
+      key: C
+    - content: None of the above
+      key: D
+- uuid: f4abb691-9853-425c-b241-f72cfebfa277
+  type: SingleChoice
+  content: Futures are regularly traded on
```

Comment:
We should ask about the `first exchange`

Reply:
Done